### ISSUE TITLE
[Enhancement] Get all tickers regardless of assetType

### ISSUE THREAD

**Issue by dtandersen** [hydrosquall/tiingo-python#338]
Would be nice to have a way to get rid of that if statement. I'm downloading supported_tickers.zip three times to get everything. Not too terrible since it's infrequent.
```
return [row for row in reader
                if row.get('assetType') == assetType]
```

**Comment by n1rna** [comment]
I suggest the `list_tickers` function can be refactored to download `supported_tickers.zip` only one time and store it inside a static directory. the existence of the file and the directory can be checked and if it fails, then we go for downloading the file.

**Comment by hydrosquall** [comment]
Hi @dtandersen, thanks for the suggestion and @n1rna for the suggested approach. I think that caching the downloaded file is one way to do it. However, we then run into the issue of needing to cache-bust the downloaded file, and I don't have a ready solution yet. Another idea is to store the downloaded file in memory for the lifetime of the TiingoClient instance, this assumes that people don't have a TiingoClient running for days on end. However, that would increase the memory pressure of running TiingoClient, which I'm not sure is a worthwhile tradeoff yet.

In the short term, what I will do is modify the listTickers function to make the `assetType` parameter optional, let me know if this will address your use-case @dtandersen!

**Comment by hydrosquall** [comment]
For any visiting Hacktoberfest participants, the ask here has 2 parts:

1. Modify the `list_tickers` method so that `assetType` is optional. If the user doesn't pass in an `assetType`, all rows should be returned without applying a filter. One way to do this is by using a Python default argument.
2. Add a unit test right above the existing `test_list_stock_tickers` function, which tests that the `list_tickers` function works when called without any arguments.

If this doesn't get picked up in October, I'll put this in the next release in November.

**Comment by n1rna** [comment]
@hydrosquall I would like to work on this issue.
I will also suggest that the `assetType` can be modified to accept list of asset types too.
so, if user does not pass in anything for `assetType` we will assume the user wants all the rows. and if it is an array, the `list_tickers` function will return all the rows that match one of asset types inside `assetType`.

**Comment by hydrosquall** [comment]
Hi @n1rna - that's awesome, go for it!

> I will also suggest that the assetType can be modified to accept list of asset types too.

This is a great idea!

**Comment by n1rna** [comment]
Good!
will get the PR ready in 1-2 days ;)

**Comment by hydrosquall** [comment]
Solved by #346 , thanks @n1rna !

**Comment by hydrosquall** [comment]
@dtandersen , you can now get the functionality requested by calling

```python
client.list_tickers()
```

without passing in any arguments.
